### PR TITLE
TELCODOCS-243: D/S Docs & RN: METAL-6 (MPHARDWARE-4/KNIDEPLOY-2108) BIOS Settings Configuration (set values)

### PR DESCRIPTION
**Pull request by johnwilkins**
Adds release notes for [TELCODOCS-243](https://issues.redhat.com/browse/TELCODOCS-243). The included hyperlink isn't showing, but the resource is included in the preview for [TELCODOCS-243](https://issues.redhat.com/browse/TELCODOCS-243) at: https://deploy-preview-41947--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/bare-metal-configuration.html  I believe https://github.com/openshift/openshift-docs/pull/41947 needs to get cherry-picked to 4.10.

Fixes: [TELCODOCS-243](https://issues.redhat.com/browse/TELCODOCS-243)

See https://issues.redhat.com/browse/TELCODOCS-243 for additional details.

Preview URL: https://deploy-preview-42533--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-host-firmware-settings

For release(s): 4.10
Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -568,6 +568,18 @@ The following enhancements to MetalLB and the MetalLB Operator are included in t
 
 For more information, see xref:../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
 
+
+[id="ocp-4-10-host-firmware-settings"]
+==== Support for modifying host firmware settings
+
+{product-title} supports the `HostFirmwareSettings` resource and the `FirmwareSchema` resource. When deploying OpenShift Container Platform on bare metal hosts, there are times when you need to make changes to the host either before or after provisioning. This can include inspecting the host's firmware and BIOS details. There are two new resources that you can use with the Bare Metal Operator (BMO):
```
Comment (bmcelvee) [inline release_notes/ocp-4-10-release-notes.adoc:575]: Suggestion: 
{product-title} supports the `HostFirmwareSettings` and `FirmwareSchema` resources.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -568,6 +568,18 @@ The following enhancements to MetalLB and the MetalLB Operator are included in t
...
+
+* `HostFirmwareSettings`: You can use the `HostFirmwareSettings` resource to retrieve and manage the BIOS settings for a host. The resource contains the complete BIOS configuration returned from the baseboard management controller (BMC). Whereas, the firmware field in the `BareMetalHost` resource returns three vendor-independent fields, the `HostFirmwareSettings` resource typically comprises many BIOS settings of vendor-specific fields per host model.
+
+* `FirmwareSchema`: You can use the `FirmwareSchema` to identify host's modifiable BIOS values and limits when making changes to host firmware settings.
```
Comment (bmcelvee) [inline release_notes/ocp-4-10-release-notes.adoc:579]: s/identify host's/identify the host's

**Comment by bmcelvee** [comment]
Left a few suggestions and asked about the cherry picking in related PR. Once that cherry pick is resolved, the Travis checks are running, the review updates are made, and SME signs off, this PR will be good to go

**Comment by johnwilkins** [comment]
@bfournie Can you review this release note and give an LGTM if it's okay?

**Comment by johnwilkins** [comment]
https://github.com/openshift/openshift-docs/pull/42695 replaces this PR, which is closed.

**Pull request closed without merging**